.npmignore

```diff
@@ -0,0 +1,11 @@
+.vscode/
+.history
+logs
+*.log
+npm-debug.log*
+.DS_Store
+*.swp
+yarn-error.log
+node_modules
+build
+dist
```

Comment:
I miss here tests and maybe some of the root files too if you want, although in Volto we are not that exhaustive.